src/MenuGraphr.ts

```diff
@@ -598,12 +604,22 @@ export class MenuGraphr implements IMenuGraphr {
             }
         }
 
-        menu.selectedIndex = selectedIndex;
```

Comment:
So would you rather I just put `[0, 0]` and remove it from the declarations at the top of the function?